backend/cdkproxymain.py

```diff
@@ -20,7 +21,7 @@
     f"Application started for envname= `{ENVNAME}` DH_DOCKER_VERSION:{os.environ.get('DH_DOCKER_VERSION')}"
 )
 
-
+load_modules(modes=[ImportMode.CDK])
```

Comment:
[Clarification] - for the case of cdkproxy we import each of the stacks. Then in the next line "registered_stacks()" we are regisstering all the imported stacks

Reply:
Yes, we import `__init__.py` for cdk module, which imports all stacks and policies. Then stacks register themselves using `@stack` decorator (just re-used the current implementation)